lib/forms/CoBrALab-Mouse-MRI-Form/index.ts

```diff
@@ -0,0 +1,368 @@
+/* eslint-disable perfectionist/sort-objects */
+
+import { defineInstrument } from '/runtime/v1/@opendatacapture/runtime-core'
+import { z } from '/runtime/v1/zod@3.23.x'
+
+
+const scanNameOptions = {
+  "Localizer": "Localizer",
+  "T1_FLASH_3D_100iso_10deg":"T1_FLASH_3D_100iso_10deg",
+  "FLASH_3D_reduced":"FLASH_3D_reduced",
+  "ADJ_B0MAP":"ADJ_B0MAP",
+  "revNoTrigAdjPEOffEPI":"revNoTrigAdjPEOffEPI",
+  "B0 Map":"B0 Map",
+  "B1map_RARE_60deg_4s":"B1map_RARE_60deg_4s",
+  "B1map_RARE_120deg_4s":"B1map_RARE_120deg_4s",
+  "MGE_MTOff":"MGE_MTOff",
+  "MGE_MTOn":"MGE_MTOn",
+  "MGE_MTOff_Tw1_30deg":"MGE_MTOff_Tw1_30deg",
+  "T2star_FID_EPI_sat_dan_ver_original":"T2star_FID_EPI_sat_dan_ver_original",
+  "exvivoDanFLASH":"exvivoDanFLASH"
+
+}
+
+
+export default defineInstrument({
+  kind: 'FORM',
+  language: 'en',
+  tags: ['Mouse', 'MRI', 'Structural', 'fMRI'],
+  internal: {
+    edition: 1,
+    name: 'MOUSE_MRI_FORM'
+  },
+  content: {
+    mriOperatorName: {
+        kind: "string",
+        variant: "input",
+        label: "Name of MRI operator"
+    },
+    coilType: {
+        kind: "string",
+        variant: "radio",
+        label: "Type of coil",
+        options: {
+            "Cryocoil":"Cryocoil",
+            "23mm volumetric": "23mm volumetric"
+        }
+    },
+    paravisionVersion: {
+        kind: "string",
+        variant: "radio",
+        label: "Paravision version",
+        options: {
+            "PV6":"PV6",
+            "PV5": "PV5"
+        }
+    },
+
+    scanRecordInfo: {
+      kind: "record-array",
+      label: "MRI scan record",
+      fieldset: {
+        mriScanName: {
+          kind: "string",
+          variant: "select",
+          label: "Scan name",
+          options: scanNameOptions
+        },
+        exVivoScan: {
+          kind: "boolean",
+          variant: "radio",
+          label: "Was scan done on ex-vivo subject?"
+        },
```

Comment:
this is a subject-level, not a scan-level question

in/ex vivo and in/ex craino only need to be asked once at the top

Reply:
All the physiology monitoring and anaesthetic can be suppressed if this is ex-vivo scanning.